tests/test_bgp_af.rb

```diff
@@ -314,65 +371,59 @@ def test_additional_paths_selection
     #
     # Set and verify
     #
+    if platform == :ios_xr
+      config_ios_xr_dependencies(asn)
+      config('route-policy drop_all', '#drop_all', 'end-policy')
```

Comment:
I think it would probably be fine to just remove the '#drop_all' arg from the config call.